addons/calendar/static/src/xml/base_calendar.xml

```diff
@@ -2,7 +2,9 @@
 <template>
     <t t-name="Many2ManyAttendeeTag" t-extend="FieldMany2ManyTag">
```

Comment:
you should extend FieldMany2ManyTagAvatar and not repeat
<img t-attf-src="/web/image/#{avatarModel}/#{el.id}/#{avatarField}" class="rounded-circle"/>

Reply:
I tried it first but it doesn't work, maybe I am not sure but qweb engine does not support as FieldMany2ManyTagAvatar also extended view by FieldMany2ManyTag

Reply:
I think
```xml
    <t t-name="Many2ManyAttendeeTag" t-extend="FieldMany2ManyTagAvatar">
        <t t-jquery="span:first" t-operation="prepend">
            <span t-if="attendeesData[el_index] and attendeesData[el_index].status and attendeesData[el_index].status !== 'needsAction'"
                t-attf-class="o_calendar_invitation #{attendeesData[el_index].status}"/>
        </t>
    </t>
```
works

Reply:
Thanks @Polymorphe57 It works